src/main/java/core/basesyntax/Circle.java

```diff
@@ -0,0 +1,21 @@
+package core.basesyntax;
+
+public class Circle extends Figure implements AreaCalculator, Drawable {
```

Comment:
Если класс Figure имплементит интерфейсы AreaCalculator и Drawable, то все его наследники делают это автоматически (исправь во всех классах)
```suggestion
public class Circle extends Figure {
```